src/game/ProgressBar.svelte

```diff
@@ -0,0 +1,45 @@
+<script>
+    export let error;
+    export let width;
```

Comment:
```suggestion
    export let progress;
```

Un nom plus fonctionnel que width